qemu/tests/cfg/resize_raw_img.cfg

```diff
@@ -0,0 +1,22 @@
+- resize_raw_img:
+    only raw
+    virt_test_type = qemu
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    type = resize_raw_img
+    images = "test"
+    image_format_test = raw
+    image_name_test = images/test
+    variants:
+        - increase_size:
+            # These two params must be provided to calculate size changes.
+            # You can change size_changes based on your cases.
+            image_size_test = 10G
+            size_changes = "+1G"
+        - shrink_size:
+            required_qemu = [2.12.0, )
+            # These two params must be provided to calculate size changes.
+            # You can change size_changes based on your cases.
+            image_size_test = 10G
```

Comment:
I would suggest to define one single `image_size_test` outside the variants, since currently they are same, and thus you can have the above comments only explained once, which looks redundant now. Like:
```
        # These two params must be provided to calculate size changes.
        # You can change size_changes based on your cases.
        image_size_test = 10G
        size_changes = "+1G"
        variants:
            - increase_size:
            - shrink_size:
                required_qemu = [2.12.0, )
                size_changes = "-2G"
```